common/types/config.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { schema, TypeOf } from '@osd/config-schema';
+
+export const configSchema = schema.object({
+  enabled: schema.boolean({ defaultValue: true }),
```

Comment:
Since our plugin was experimental, shall we set this default value to `false`?
It seems this `enabled` was new added, shall we add to the `opensearch-build` repo?

Reply:
your call! imo i think true is fine. because this is the difference between the files not being loaded into the server or not. 

this i used to verify my code if not available if plugins still work with the component. and we can tell infra team if they want to fully disabled it and add it to the env variables. but for development it might be weird to install a plugin and configure your files to be true otherwise your files wont be loaded.